src/main/java/TaskManager.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public ArrayList<Task> Tasks;
+    public TaskManager() {
+        this.Tasks = new ArrayList<Task>();
+    }
+
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINEBREAK = "____________________________________________________________";
+    public void storeTasks(String text) {
+        System.out.println(LINEBREAK);
+        String[] result = text.split(" ");
+        String[] result2 = text.split("/by ");
+        String[] result3 = text.split("/at ");
+        Todo newTodo = null;
+        Deadline newDeadline = null;
+        Event newEvent = null;
+        if (result[0].equals("todo")) {
+            try {
+                String x=result[1];
+            }
+            catch (IndexOutOfBoundsException e) {
+                System.out.println("Much ado about nothing");
+                return;
+            }
+            newTodo = new Todo(result[1]);
```

Comment:
You can try to extract out parsing things to a separate method and improve SLAP here